sdk/core/azure-core-amqp/inc/azure/core/amqp/internal/message_receiver.hpp

```diff
@@ -3,6 +3,7 @@
 
 #pragma once
 
+#include "amqp_settle_mode.hpp"
```

Comment:
```suggestion
#include "azure/core/amqp/internal/amqp_settle_mode.hpp"
```

Reply:
Why use an absolute path to the file when it's in the same directory as the header?

There doesn't seem to be any reason to force a full path on headers, especially if they're in the same directory.

Reply:
If they are installed, I think there could be cases when it makes difference between local file in the same directory, and the installed location.
We use the absolute pattern everywhere anyways, so at least for consistency.

Reply:
My point is that we don't appear to do that. There are multiple places where we include files with relative paths.